doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -150,16 +158,16 @@ analysis on inductive or coinductive objects (see :ref:`variants`).
          goal, then :n:`destruct @ident` behaves like
          :tacn:`intros` :n:`until @ident; destruct @ident`.
 
-       + If :n:`@ident` is no longer dependent in the
-         goal after application of :n:`destruct`, it is erased. To avoid erasure,
-         use parentheses, as in :n:`destruct (@ident)`.
+       + If :n:`@ident` is not referenced directly or indirectly in the
+         goal or unselected hypotheses after application of :n:`destruct`,
+         it is erased. To avoid erasure, use parentheses, as in :n:`destruct (@ident)`.
```

Comment:
```suggestion
       + If :n:`@ident` is not referenced directly or indirectly in the
         goal or unselected hypotheses after application of :n:`destruct`,
         it is erased (unless it is a section variable). To avoid erasure,
         use parentheses, as in :n:`destruct (@ident)`.
```